### ISSUE TITLE
Order Handling

### ISSUE THREAD

**Issue by cbj4074** [indiehd/web-api#20]
Key requirements:

https://github.com/indiehd/web-api/milestone/11

Related Issues:

- [x] #92 
- [x] #94  
- [x] #95

**Comment by cbj4074** [comment]
Let's talk a bit about how a Shopping Cart instance is bound to an individual shopper.

Needless to say, a customer doesn't need to have an Account to be able to checkout. In other words, customers may be "guests" when they checkout.

Does it make the most sense for a Cart instance should be bound to a session? Or is there a more elegant means by which to associate a shopper with a Cart instance?

**Comment by cbj4074** [comment]
Following an extensive side-channel discussion, we've decided that this issue, the related issues, and the parent milestone are really more about an "Order" than a "Shopping Cart".

Accordingly, we've renamed the above, as appropriate.

The fundamental distinction to be made here is that the Shopping Cart will be handled entirely on the front-end, and all we're concerned with here is handling its contents and baking them into an Order with individual Order Items.

I'll refactor the existing references to reflect this change in terminology.
